lib/modulesync/git.rb

```diff
@@ -76,8 +77,22 @@ def self.update(name, files, message, branch, bump=false, changelog=false, tag=f
         end
       end
       begin
-        repo.commit(message)
-        repo.push('origin', branch)
+        branch = "#{branch_prefix}#{branch}"
+        opts_commit = {}
+        opts_push = {}
+        if amend
+          opts_commit = {:amend => true}
+          message = nil
```

Comment:
Do we need to set this to nil? I've had a lot of cases where amending the commit also means changing the commit message.